recipe/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "sis-extras" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/sis_extras-{{ version }}.tar.gz
+  sha256: e7e1892ab5eef36e38880dc9b636597dce83162df4d51f3f7f509a7e792e8a73
+  patches:
+    - patches/0001-loosen-Python-and-streamlit-constraints.patch
+    - patches/0002-add-a-timeout-to-the-test-run.patch
+
+build:
+  # Strictly, upstream is py==38 and we don't have
+  # snowflake-snowpark-python for 3.12
+  # no streamlit on s390x
+  skip: True  # [py>311 or s390x]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  build:
+    - patch     # [not win]
+    - m2-patch  # [win]
+  host:
+    - python
+    - pdm-backend
+    - pip
+  run:
+    - python
+    # Strictly, upstream is streamlit==1.31.1
+    - streamlit
+    - snowflake-snowpark-python >=1.13.0
+    - plotly >=5.20.0
+
+test:
+  source_files:
+    - tests
+  imports:
+    - sis_extras
+  requires:
+    - pip
+    - pytest
+    # sqlparse can be used if available
+    - sqlparse
+  commands:
+    - pip check
+    - pytest -v tests/test_sis_extras.py
+    - python -m tests.test_example_app
```

Comment:
@ifitchet Could you please patch the test up by setting `default_timeout=30` [here](https://github.com/blackary/sis-extras/blob/v0.1.3/tests/test_example_app.py#L3):
`at = AppTest.from_file("app.py", default_timeout=30)`

See also https://discuss.streamlit.io/t/apptest-test-build-interactive-python-data-apps-faster/54472/3

Reply:
Yep, @lorepirri pointed me there.
(I need to brush up on my finding solutions-fu.)